src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -0,0 +1,501 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace PowerAccent.Core
+{
+    using System;
+    using PowerToys.PowerAccentKeyboardService;
+
+    public enum Language
```

Comment:
Numbers (¹, ², ...) are missing as own language group.

Reply:
And "Special keys" for "VK_Comma".

Reply:
What do you mean by language group? Do you mean for each language except `ALL` ?

Reply:
You can select `each language` or `currency` or `all`. But you can't select only the `numbers` or only the signs on `VK_Comma`.

Reply:
That means create a section Math, & a section special characters (as special characters contain a lot of Math elements, I'm sure having 2 is necessary). It is something we need to think a bit better I guess (I don't know what are the others thinking about that, but having a Math section is may be too much).

I invite you to create a specific issue to discuss in it. If globally people are agree with that, may be we can implement it.

Reply:
The only idea behind the code comment was that we have `currency` but not the other two I mentioned.

Reply:
Numbers could also be a common thing for all languages maybe? So whatever you select, numbers could be there?

Anyway, let's leave this for additional PR